TYPO3.Flow/Classes/TYPO3/Flow/Http/Client/CurlEngine.php

```diff
@@ -148,9 +148,10 @@ public function sendRequest(Http\Request $request)
         curl_close($curlHandle);
 
         $response = Http\Response::createFromRaw($curlResult);
-        if ($response->getStatusCode() === 100) {
+        while (substr($response->getContent(), 0, 5) === 'HTTP/' || $response->getStatusCode() === 100) {
```

Comment:
Is that safe to check the response _content_ to start with "HTTP/"?
Seems a bit "magic" to me. Wouldn't the `$curlResult` contain more details about this so we can catch that case more "safely"?